src/CMakeLists.txt

```diff
@@ -64,10 +64,7 @@ trilean_option(ALICEVISION_USE_UNCERTAINTYTE "Enable Uncertainty computation" AU
 trilean_option(ALICEVISION_USE_CUDA "Enable CUDA" ON)
 trilean_option(ALICEVISION_USE_OPENCV "Build opencv+aliceVision samples programs" OFF)
 
-# Since OpenCV 3, SIFT is no longer in the default modules. See
-# https://github.com/itseez/opencv_contrib for more informations.
-# Enable this to be able to use OpenCV SIFT in main_ComputeFeatures_OpenCV.
-option(ALICEVISION_USE_OCVSIFT "Add or not OpenCV SIFT in available features" OFF)
+option(ALICEVISION_USE_OCVSIFT "Add or not OpenCV SIFT in available features" ON)
```

Comment:
we need to raise the requirement to the version of ocv that will sport the changes (3.4.X or 4.X which are not out yet).

Also they are planning to move it inside the main repository (https://github.com/opencv/opencv/wiki/GSoC_2020#idea-better-sift-in-the-main-repository), at that point we can just get rid of this option because it will be built by default anyway

Reply:
Yes, but in the meantime we could just enable this by default without raising the ocv version.

Reply:
We don't use it by default. It's only interesting for developers to be able to compare different SIFT implementations.
From our experiments, the opencv SIFT implementation is not as good as vlfeat (cpu) / popsift (gpu).

What we need to adjust is the message in https://github.com/alicevision/AliceVision/blob/develop/COPYING.md.

Reply:
```cmake
mark_as_advanced(FORCE ALICEVISION_USE_OCVSIFT)
```
https://cmake.org/cmake/help/v3.12/command/mark_as_advanced.html